### PR TITLE
feat(audio): batch 3 of native updates

### PR DESCRIPTION
**Pull request by AvarianKnight**

### CODE REVIEW

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Looks good so far, you could consider this nitpicking, albeit I'm somewhat sure the alias for that other native is required.

File: AUDIO/FindRadioStationIndex.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ ns: AUDIO
 int FIND_RADIO_STATION_INDEX(int stationNameHash);
 ```
 
-
 ## Parameters
-* **stationNameHash**: 
+* **stationNameHash**: the station name hash to search for
 
 ## Return value
+The station index for the specified hash, or ?? otherwise.
```
[the remark this answers is not in the document]
Comment (AvarianKnight) [reply AUDIO/FindRadioStationIndex.md:15]: This was meant to be replaced by whatever the game has as the default

[the remark this answers is not in the document]
Comment (4mmonium) [reply AUDIO/FindRadioStationIndex.md:15]: Hey there, the game returns `254` if not found.

image

File: AUDIO/GetAmbientVoiceNameHash.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
 ns: AUDIO
 aliases: ["0x5E203DA2BA15D436"]
```
[the remark this answers is not in the document]
Comment (AvarianKnight) [reply AUDIO/GetAmbientVoiceNameHash.md:3]: Do natives that keep the same name need to have an alias?

I don't think the underscore ever gets used anywhere for an alias to be necessary, please correct me if I'm wrong

[the remark this answers is not in the document]
Comment (4mmonium) [reply AUDIO/GetAmbientVoiceNameHash.md]: > Do natives that keep the same name need to have an alias?
> 
> I don't think the underscore ever gets used anywhere for an alias to be necessary, please correct me if I'm wrong

That's a great question, I'm going to ask. Anyway, this is what I have found:
https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/CLOCK/GetUtcTime.md?plain=1#L2-L6

https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/DECORATOR/DecorSetFloat.md?plain=1#L2-L6

https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/ENTITY/GetAnimDuration.md?plain=1#L2-L6

https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/ENTITY/GetEntityCollisionDisabled.md?plain=1#L2-L6

https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/STATS/PlaystatsCrateCreated.md?plain=1#L2-L6

https://github.com/citizenfx/natives/blob/4ec7d47d236ec30b8861b1e693ee521ef807274f/STATS/PlaystatsHeistSaveCheat.md?plain=1#L2-L6

[the remark this answers is not in the document]
Comment (4mmonium) [reply AUDIO/GetAmbientVoiceNameHash.md]: @AvarianKnight Got news from thorium, apparently yes they do need any preceding name as an alias as that'll make sure the natives and enums still have them present.

Hope this clears it up for you 😊

**Pull request merged**